Convert Routes to Native Classes and hide unfinished content (#638)
ember-learn/guides-source: guides/release/getting-started/quick-start.md

you will have access to a new `ember` command in your terminal.
You can use the `ember new` command to create a new application.

<!-- needs-octane-release-update -->
```bash

> ember new ember-quickstart -b @ember/octane-app-blueprint

Reviewer: This fixes some of #633

The `action` helper allows you to add event listeners to elements and call named functions.
By default, the `action` helper adds a `click` event listener,
but it can be used to listen for any element event.
Now, when the `li` element is clicked, a `showPerson` method will be called in the `people-list` component.

Add the action to the `people-list.js` file:

> ```javascript {data-filename="app/components/people-list.js" data-diff="+4,+5,+6,+7,+8"}

Reviewer: ```suggestion
```javascript {data-filename="app/components/people-list.js" data-diff="+2,+5,+6,+7,+8"}
```

to represent

```js
import Component from '@glimmer/component';
import { action } from '@ember/object'; // 2

export default class PeopleList extends Component {
  @action // 5
  showPerson(person) { // 6
    alert(person); // 7
  } // 8
}
```
being added to the file at this step

}
```

> The `@action` is something called a decorator, and it lets your Ember app know that the `showPerson` method should be available for the template to use.

Reviewer: ```suggestion
The `@action` is something called a decorator, and it lets your Ember app know that the `showPerson` method should be invocable by name in the component's template via `{{action "showPerson"}}`
```
Suggested rewording to avoid implying that we'd be unable to use find this method via `{{this.showPerson}}` even without the `@action` decorator
